tensix/CMakeLists.txt

```
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.10)
+
+project(tensix-isa-simulator
+    LANGUAGES CXX
```

Since this is just a wrapper for sub-projects, we don't need to toolchains for this project.

```suggestion
    LANGUAGES NONE
```